tests/unit/test_utils.py

```diff
@@ -0,0 +1,94 @@
+# MIT License
+#
+# Copyright (c) 2018-2019 Red Hat, Inc.
```

Comment:
2020?

Reply:
Please, create a new (good-first?) issue -- it's in all files..;(